tests/integration/targets/lightsail_static_ip/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+- module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key | default(omit) }}'
+      aws_secret_key: '{{ aws_secret_key | default(omit) }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region | default(omit) }}'
+
+  block:
+
+    # ==== Tests ===================================================
+
+    - name: Create a new static IP
+      lightsail_static_ip:
+        name: "{{ static_ip_name }}"
+        region: "{{ region }}"
+      register: result
+
+    - assert:
+        that:
+          - result.changed == True
+          - "'static_ip' in result and result.static_ip.name == static_ip_name"
+          - "'static_ip' in result and result.static_ip.resourceType == 'StaticIP'"
```

Comment:
This assertion fails, the key and value being returned are `"resource_type": "StaticIp",`